Blockzilla/DomainCompletion.swift

```diff
@@ -7,7 +7,13 @@ import AutocompleteTextField
 
 class DomainCompletion: AutocompleteTextFieldCompletionSource {
     private lazy var topDomains: [String] = {
-        let filePath = Bundle.main.path(forResource: "topdomains", ofType: "txt")
+        var filePath = Bundle.main.path(forResource: "topdomains", ofType: "txt")
```

Comment:
I wonder if this can be done in a more generalized way and without an if statement. Name all top domains files with the locale as a suffix (just like localizations) so then you can more straightforwardly write this code. The default if there is no localization is the English one, as it is now.